README.md

```diff
@@ -309,6 +309,69 @@ vim examples/kubernetes/10_modbus-proxy.configmap.yaml
 kubectl apply -f examples/kubernetes
 ```
 
+## Installing modbus-proxy on Raspberry PI VENV (virutal environment)
+
+Create subfolder for all virutal environments and activate venv 'modbusproxy'
+```bash
+sudo mkdir python
+cd python
+python3 -m venv modbusproxy
+source modbusproxy/bin/activate
+```
+It should look like this now:
+```bash
+(modbusproxy) pi@raspberrypi:~/python $
+```
+
+Install modbus-proxy and create config yaml
+```bash
+pip3 install modbus-proxy
+pip3 install modbus-proxy[yaml]
```

Comment:
No need for `pip3` and no need for the first install. 
Should be `pip install modbus-proxy[yaml]`